config/packages/doctrine.yaml

```diff
@@ -28,6 +28,8 @@ doctrine:
         report_fields_where_declared: true
         validate_xml_mapping: true
         naming_strategy: doctrine.orm.naming_strategy.underscore_number_aware
+        identity_generation_preferences:
+            Doctrine\DBAL\Platforms\PostgreSQLPlatform: identity
```

Comment:
This seems to be a new feature in Doctrine. Which will add support for Identity Column I believe. https://neon.tech/postgresql/postgresql-tutorial/postgresql-identity-column